package.json

```diff
@@ -70,8 +70,10 @@
     "request": "2.67.0",
     "serve-static": "1.10.0",
     "time-grunt": "1.2.1",
-    "ua-parser": "0.3.5",
-    "universal-analytics": "0.3.9"
+    "uap-core": "git://github.com/ua-parser/uap-core.git#1f509812",
+    "uap-ref-impl": "0.2.0",
+    "universal-analytics": "0.3.9",
+    "yamlparser": "0.0.2"
```

Comment:
> We should really make a module out of this code update (something like https://github.com/fedot/node-uap) because we use ua-parser in at least one other fxa server (that also sends data to DataDog)

Hey, I didn't see that package. We might as well just use that if the deps are the same.